migrations/20200327213301_create_patients_table.js

```diff
@@ -0,0 +1,68 @@
+
+exports.up = function (knex) {
+  // Create new patient table
+  return knex.schema.createTable("patients", (table) => {
+    table.increments("id").primary();
+    table.text("name");
+    table.text("phone_number");
```

Comment:
We should be consistent with casing. Looks like the current convention is camelCase

Reply:
Also patient_id

Reply:
:+1: